library/macos/FLETextInputPlugin.mm

```diff
@@ -0,0 +1,390 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FLETextInputPlugin.h"
+
+#import "FLEJSONMethodCodec.h"
+#import "FLEViewController+Internal.h"
+
+#include "../common/internal/text_input_model.h"
+
+#include <iostream>
+#include <map>
+#include <memory>
```

Comment:
These go after the associated header but before all the other headers (per style).

Also, why are iostream and map here?